src/tags/if_block.rs

```diff
@@ -336,4 +350,75 @@ mod test {
         let output = template.render(&mut context).unwrap();
         assert_eq!(output, Some("fourth".to_owned()));
     }
+
+    #[test]
+    fn contains_comparison() {
```

Comment:
Personally, I prefer the tests to be broken up by their purpose.

This is helpful for the person modifying tests and the person analyzing a test failure.

So for example,
```rust
#[test]
fn contains_string_literals() {
   ...
}

#[test]
fn contains_else() {
   ...
}

#[test]
fn test contains_variable() {
   ...
}
```

(I don't recommend literally copying my function names above when splitting this function because I'm making guesses as to your intent.  You know what you intended with each test and this serves as a way for you to document it)